LambdaEngine/Source/Resources/ResourceLoader.cpp

```diff
@@ -214,7 +216,6 @@ namespace LambdaEngine
 			aiProcess_ImproveCacheLocality		|
 			aiProcess_LimitBoneWeights			|
```

Comment:
Could be necessary to keep this number(default = 4) in mind so we don't renormalize every bone weight if the imported Model has more weights than 4 applied to it.